salt/states/file.py

```diff
@@ -2332,9 +2332,17 @@ def managed(name,
                 )
 
                 if salt.utils.is_windows():
-                    ret = __salt__['file.check_perms'](
-                        name, ret, win_owner, win_perms, win_deny_perms,
-                        win_inheritance)
+                    try:
+                        ret = __salt__['file.check_perms'](
+                            path=name,
```

Comment:
@twangboy I don't think this way of calling the options in the function will work on 2017.7: https://github.com/saltstack/salt/blob/2017.7/salt/modules/file.py#L4180

Reply:
This won't actually call the `check_perms` function in `salt/modules/file.py` as it has been namespaced in `salt/modules/win_file.py`

Reply:
Ohhh OK. thank you. I was thinking that might be the case but I wasn't sure.